base.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:stretch
+
+RUN echo "deb http://http.debian.net/debian stretch-backports main" | \
+     tee --append /etc/apt/sources.list.d/stretch-backports.list > /dev/null && \ 
+     apt-get update -y && \
+     apt-get install -y \
+     wget \
+     curl \
+     ruby \
+     ruby-dev \
+     rubygems \
+     build-essential \
+     git-core && \
+     apt-get install -y -t stretch-backports openjdk-8-jdk && \
+     update-java-alternatives -s java-1.8.0-openjdk-amd64 && \
+     apt-get upgrade -y
+
+RUN export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+RUN gem install fpm
+RUN fpm --version
+
+#Installing Apache Ant
+RUN wget http://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.gz
```

Comment:
Do you need this particular version of ant? You could consider having the ant version in your `versions.json` and tagging the ant image with that version number.

Reply:
This was the version of ant I installed originally, I haven't really tested any other versions

Reply:
The main reason I said this is it feels weird that you're rebuilding the ant image based on the solr major version, when it has no dependencies on that. Realistically you probably want to be rebuilding this image every time you redeploy to pick up security fixes.

Reply:
🤷 I'm not entirely sure, I can't remember how I installed ant on my machine _(unless it was pre installed)_, and back in the day before jenkins was updated, we had a node which just had things hard installed on there , so I had to spend time just trying to figure out how to install ant on a docker image to build jenkins

Reply:
Every merge may be a bit too keen actually if this repo is updated frequently.

Reply:
Following your other comment, I'm re-working this to make a simpler ant image, and having a parameter for versions, not sure which version is best to use, but will stick with the current version we've been running for now _(but ofc, has the option to change in future)_